UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -545,7 +544,7 @@ private void Add_Field_OnClick(object sender, RoutedEventArgs e)
                             {
                                 RaiseAlert(
                                     "Class Creation Failed",
```

Comment:
Window title for alert dialog for creating fields says "Class Creation Failed"

Reply:
Shucks.  I'm on it!